selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/SeleniumWebDriverHelper.java

```diff
@@ -899,4 +903,58 @@ public Actions getAction(SeleniumWebDriver seleniumWebDriver) {
   public Actions getAction() {
     return getAction(this.seleniumWebDriver);
   }
+
+  /** Switches to IDE frame and waits for Project Explorer is available. */
+  public void switchToIdeFrameAndWaitAvailability() {
+    switchToIdeFrameAndWaitAvailability(APPLICATION_START_TIMEOUT_SEC);
+  }
+
+  /**
+   * Switches to IDE frame and waits during {@code timeout} for Project Explorer is available.
+   *
+   * @param timeout waiting time in seconds
+   */
+  public void switchToIdeFrameAndWaitAvailability(int timeout) {
+    webDriverWaitFactory
+        .get(timeout)
+        .until(
+            (ExpectedCondition<Boolean>)
+                driver -> {
+                  waitAndSwitchToFrame(By.id("ide-application-iframe"), PREPARING_WS_TIMEOUT_SEC);
+                  if (isVisible(By.id("gwt-debug-projectTree"))) {
+                    return true;
+                  }
+
+                  seleniumWebDriver.switchTo().parentFrame();
+                  return false;
+                });
+  }
+
+  /** Waits while in a browser appears more than one window */
+  public void waitOpenedSomeWin() {
+    webDriverWaitFactory
+        .get(WIDGET_TIMEOUT_SEC)
+        .until(
+            (ExpectedCondition<Boolean>)
+                input -> {
+                  Set<String> driverWindows = seleniumWebDriver.getWindowHandles();
+                  return (driverWindows.size() > 1);
+                });
+  }
+
+  /**
+   * Switches to next browser window (this means that if opened 2 windows, and we are in the window
+   * 1, we will be switched into the window 2)
+   *
+   * @param currentWindowHandler
+   */
+  public void switchToNoneCurrentWindow(String currentWindowHandler) {
```

Comment:
Lets rename it to make the method propose more obvious: `switchToNextWindow()`. 
Argument `currentWindowHandler` is also misleading so as it seems could be any non-current window handler. So, lets rename the argument to be more precise, for example: `windowHandlerToSwitchFrom`.